package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sparkfabrik/capacitor-plugin-idfa",
-  "version": "4.0.1",
```

Comment:
please remove this change as the version will be set on release

Reply:
Done. Sorry.

Reply:
thanks! no problem